source/data/History.rst

```diff
@@ -5,41 +5,40 @@
 History
 -------
 
-We need the `/history`_ endpoint to:
+TradingView needs the `/history`_ endpoint to:
 
 * fill the database with deep history,
-* compensate data from the `/streaming`_ in case of problems.
+* compensate data from `/streaming`_ in case of problems.
 
-After initial filling of the database we make regular requests to `/history`_ endpoint to keep the data relevant. Our 
-data feed requests 1-minute bars for the whole day per request. Requests are made sequentially from the current time 
-to the past. 
+After initial filling of the database, TradingView makes regular requests to `/history`_ endpoint to keep the data relevant.
+TradingView data feed requests 1-minute bars for the whole day per request.
+Requests are made sequentially from the current time to the past. 
 
-If there is no data in the requested and previous time periods then you should set the status code to ``no_data``. The
-API should respond with an empty response in case of requesting the range containing no historical data.
+Data requirements
+..................
 
-We will request `/history`_ until the date that the broker reported in the **Data requirements from**. Without this
-date, we will request a history up to 1800 year.
+Your data should meet the following requirements:
 
-.. code-block:: json
+- The data must be real, from the production environment preferably. Otherwise, integration tests will fail.
+- Real-time data obtained from the `/streaming`_ endpoint must match the historical data obtained from `/history`_. The allowed count of mismatched bars (candles) must not exceed 5% for frequently traded symbols. Otherwise, integration into TradingView is not possible.
+- The data must not include unreasonable price gaps, historical data gaps on 1-minute, daily resolutions (temporal gaps), and incorrect prices (adhesions).
+- The daily bar time should be 00:00 UTC and expected to be a trading day, not a day when the session starts.
```

Comment:
Don't know if this line is obvious for all users. Might be useful to add a small example like you've provided me in slack.